beams/beams/doctype/local_enquiry_report/local_enquiry_report.py

```diff
@@ -1,16 +1,28 @@
-# Copyright (c) 2024, efeone and contributors
+# Copyright (c) 2022, Frappe Technologies Pvt. Ltd. and contributors
 # For license information, please see license.txt
 
 import frappe
 from frappe.model.document import Document
-from frappe.utils import getdate, add_days, today
+from frappe.utils import getdate, add_days, today, nowdate
+from frappe import _
+from frappe.desk.form.assign_to import add as add_assign
+from frappe.utils.user import get_users_with_role
 
 class LocalEnquiryReport(Document):
+
     def validate(self):
-        self.information_required()
+        self.information_required()  # Assuming you need to call this once
         self.set_expected_completion_date()
-        self.information_required()
-        
+
+        """Ensure enquiry officer selection based on workflow state."""
+        # Check if the workflow state is 'Assign to Enquiry Officer'
+        if self.workflow_state == 'Assign to Enquiry Officer':
+            if not self.enquiry_officer:
+                frappe.msgprint(_("Please select an enquiry officer before proceeding."))
+            else:
+                # If the enquiry officer is selected, update the field
+                frappe.db.set_value("Local Enquiry Report", self.name, "enquiry_officer", self.enquiry_officer)
```

Comment:
What is this line supposed to do? this line is simmply setting the same value in the same field